### PR TITLE
Add Missing Type Annotations

### LINKED ISSUE #19: Missing Type Annotations
There are missing type annotations for some functions in this library.

The `typing` module does not exist on CircuitPython devices so the import needs to be wrapped in try/except to catch the error for missing import. There is an example of how that is done here:

```python
try:
    from typing import List, Tuple
except ImportError:
    pass
```

Once imported the typing annotations for the argument type(s), and return type(s) can be added to the function signature. Here is an example of a function that has had this done already:

```python
def wrap_text_to_pixels(
    string: str, max_width: int, font=None, indent0: str = "", indent1: str = ""
) -> List[str]:
```
If you are new to Git or Github we have a guide about contributing to our projects here: https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github

There is also a guide that covers our CI utilities and how to run them locally to ensure they will pass in Github Actions here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code In particular the pages: `Sharing docs on ReadTheDocs` and `Check your code with pre-commit` contain the tools to install and commands to run locally to run the checks.

If you are attempting to resolve this issue and need help, you can post a comment on this issue and tag both @foamyguy and @kattni or reach out to us on Discord: https://adafru.it/discord in the `#circuitpython-dev` channel.

The following locations are reported by mypy to be missing type annotations:

- [ ] adafruit_vcnl4010.py:109
- [ ] adafruit_vcnl4010.py:119
- [ ] adafruit_vcnl4010.py:126
- [ ] adafruit_vcnl4010.py:133
- [ ] adafruit_vcnl4010.py:150
- [ ] adafruit_vcnl4010.py:167
- [ ] adafruit_vcnl4010.py:186

### PR DESCRIPTION
**Pull request by tcfranks**
resolves #19 
submitted for review / comment

### CODE REVIEW

File: adafruit_vcnl4010.py [hunk 1]
```diff
@@ -30,6 +30,11 @@
 
 from adafruit_bus_device import i2c_device
 
+try:
+    from typing import Optional, List  # pylint: disable=unused-import
```
Comment (tekktrik) [inline adafruit_vcnl4010.py]: The pylint disable isn't needed since the typing module is being used.

File: adafruit_vcnl4010.py [hunk 2]
```diff
@@ -273,7 +278,7 @@ def ambient(self):
     # pylint: enable=inconsistent-return-statements
 
     @property
-    def ambient_lux(self):
+    def ambient_lux(self) -> int:
```
Comment (tekktrik) [inline adafruit_vcnl4010.py]: This should be `float` based on the multiplication with a `float`.

**Review by tekktrik — CHANGES_REQUESTED** [review]

File: adafruit_vcnl4010.py (hunk 2 above)
Comment (tcfranks) [reply adafruit_vcnl4010.py]: oops, should have checked the constant - sorry

File: adafruit_vcnl4010.py (hunk 1 above)
Comment (tekktrik) [reply adafruit_vcnl4010.py]: Since the imported items aren't being used:

```suggestion
    import typing  # pylint: disable=unused-import
```

**Review by tekktrik — CHANGES_REQUESTED** [review]

**Comment by tcfranks** [comment]
resubmitted with changes

**Review by tekktrik — APPROVED** [review]
Nice!

**Pull request merged**